### PR TITLE
Use routes history mode and global eslint config

### PR DESCRIPTION
**Pull request by skjnldsv**
- moved to regular url for better viduals
- use global eslint config
- cleaned up code
- updated makefile to fit what we use in our other apps

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -1,22 +1,22 @@
 <template>
-  <router-view />
+	<router-view />
 </template>
 
 <script>
-import { mutations } from './store/';
+import { mutations } from './store/'
```
Comment (marcelklehr) [inline src/App.vue]: Wut? You're doing non-semicolon style now? I like it :)

**Review by marcelklehr — APPROVED** [review]
Sweet, thanks alot! :)

File: src/App.vue (hunk 1 above)
Comment (skjnldsv) [reply src/App.vue]: Yes, we removed it a few time ago in the default eslint config :)
When I started migrating things to vue, then everyone just used my config, so in the end almost all newly migrated apps are using similar (very close) configs. Thus the new unified config ;)

**Comment by skjnldsv** [comment]
:tada: :tada: :tada:

**Pull request merged**